sig-cli/README.md

```diff
@@ -60,6 +60,12 @@ subprojects, and resolve cross-subproject technical issues and decisions.
     - [@kubernetes/sig-cli-test-failures](https://github.com/orgs/kubernetes/teams/sig-cli-test-failures) - Test Failures and Triage
 - Steering Committee Liaison: Paco Xu 徐俊杰 (**[@pacoxu](https://github.com/pacoxu)**)
 
+## Working Groups
+
+The following [working groups][working-group-definition] are sponsored by sig-cli:
```

Comment:
is SIG CLI the sponsor or SIG Node?

Reply:
We are adding this to all the stakeholder SIGs and they are in favor of participating (or at least consulting) in this effort. Btw this file is generated.